recipe/meta.yaml

```diff
@@ -10,33 +10,47 @@ source:
   sha256: c2edb73a08e9e0e6f65a0e6af18b059b8b1cdd5bef997d7a0b181df93dc81539
 
 build:
-  number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
-  skip: True  # [py<36]
-  entry_points:
-    - black = black:patched_main
+  number: 1
 
-requirements:
-  host:
-    - python
-    - pip
-    - setuptools_scm
-  run:
-    - python
-    - click >=6.5
-    - attrs >=18.1.0
-    - appdirs
-    - toml >=0.9.4
-    - typed-ast >=1.4.0
-    - regex
-    - pathspec >=0.6,<1
-    - dataclasses >=0.6  # [py<37]
-    - typing_extensions >=3.7.4
-    - mypy_extensions >=0.4.3
-
-test:
-  commands:
-    - black --help
+outputs:
+  - name: {{ name }}
+    build:
+      noarch: python
+      script: "python -m pip install . -vv"
+      entry_points:
+        - black = black:patched_main
```

Comment:
I would expect `blackd` to also be available here as it would be with the `black` python package.